apps/ai-hero/src/app/page.tsx

```
@@ -86,20 +90,94 @@ const Home = async (props: Props) => {
 		title: page.resources[0]?.resource?.fields?.title,
 	}
 	const searchParams = await props.searchParams
+	const couponCodeOrId = searchParams?.code || searchParams?.coupon
+	let validCoupon = null
+	let coupon = null
+	if (couponCodeOrId) {
+		coupon = await getCouponForCode(couponCodeOrId, [], courseBuilderAdapter)
+		if (coupon && coupon.isValid) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)